### PR TITLE
Delete program if it is removed by user

### PR DESCRIPTION
**Pull request by GinVavilon**
Fix problem: if user removes program game is not shown on launch

### CODE REVIEW

**Comment by cgutman** [comment]
Can you explain a bit more about exactly what this is fixing? What actions are required to trigger the bug?

File: app/src/main/java/com/limelight/utils/TvChannelHelper.java [hunk 1]
```diff
@@ -224,7 +223,17 @@ private Long getProgramId(long channelId, String appId) {
...
+                    long id = cursor.getLong(ID_INDEX);
+                    int browsable = cursor.getInt(PROGRAM_BROWSABLE_INDEX);
+                    if (browsable != 0) {
+                        return id;
+                    } else {
+                        int countDeleted = context.getContentResolver().delete(TvContract.buildPreviewProgramUri(id), null, null);
+                        if (countDeleted > 0) {
+                            LimeLog.info("Preview program has been deleted");
+                        }
+                        LimeLog.warning("Preview program has not been deleted");
```
Comment (cgutman) [inline app/src/main/java/com/limelight/utils/TvChannelHelper.java]: This should be in an `else` block

**Comment by GinVavilon** [comment]
> Can you explain a bit more about exactly what this is fixing? What actions are required to trigger the bug?
Steps for reproduse:
1) Launch game
2) Remove game from home screen (long press -> Remove)
3) Launch game again
Result:
Game doen'r appear in channel on home screen
Expected:
New icon of game on home screen

When user removes program from launcher. It is not removing row 
from contentprovider only mark it. And when user launch game next time program doesn't appear on channel, because moonlight updates row which is marked as removed (browsable == 0). App also doesn't have  accesess to change that column.

**Comment by cgutman** [comment]
Thanks, that makes sense.

**Pull request merged**